docs/framework/wcf/samples/binding-extensibility.md

```diff
@@ -22,5 +22,5 @@ This section contains samples that demonstrate custom binding in [!INCLUDE[indig
  <xref:System.ServiceModel.NetHttpBinding>  
  Demonstrates how to implement a binding that stacks <xref:System.ServiceModel.Channels.HttpTransportBindingElement> or the <xref:System.ServiceModel.Channels.HttpsTransportBindingElement> on top of the <xref:System.ServiceModel.Channels.BinaryMessageEncodingBindingElement>.  
   
- <!--zz <xref:System.ServiceModel.WSStreamedHttpBinding> --> `System.ServiceModel.WSStreamedHttpBinding` 
+ <xref:System.ServiceModel.WSStreamedHttpBinding> 
```

Comment:
should actually link to a topic:
`[WSStreamedHttpBinding](wsstreamedhttpbinding.md)`